include/openssl/evp.h

```diff
@@ -1616,6 +1616,11 @@ int EVP_PKEY_param_check(EVP_PKEY_CTX *ctx);
 int EVP_PKEY_private_check(EVP_PKEY_CTX *ctx);
 int EVP_PKEY_pairwise_check(EVP_PKEY_CTX *ctx);
 
+# define EVP_PKEY_get_ex_new_index(l, p, newf, dupf, freef) \
+    CRYPTO_get_ex_new_index(CRYPTO_EX_INDEX_EVP_PKEY, l, p, newf, dupf, freef)
```

Comment:
Probably we enclose each of these params in "()"

Reply:
It's not actually necessary since comma is the lowest precedence operator, and if one of the arguments has a comma, the caller has to put it in () themselves in order to pass it as a single argument. None of the other TYPE_get_ex_new_index macros put the args in ()s so I was just following that. I don't have a problem making the change if you prefer, but I'd change them all so they're consistent.